app/uk/gov/hmrc/accessibilitystatementfrontend/models/WCAGVersion.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.accessibilitystatementfrontend.models
+
+sealed trait WCAGVersion extends EnumValue {
+  val href: String
+  val version: String
+  def value = s"$version AA"
```

Comment:
I don't think we should make a change, this looks good to me, the only bit I wondered was if having the AA here might be a footgun in future for maintainers, it's also hard coded in lots of other places, but I could imagine that someone might think they can implement a WCAG22AAA by just editing this enum maybe 🤷 over thinking probably

Reply:
@oscarduignan I agree, I think I'll delete it. Initially this was being used as a whole string, e.g. pulling through "2.2 AA" into messages, but that didn't work for Welsh. I'm going to update this. The only thing is that this will mean the value in the YAML files will need to change as it's coupled to the Decoder for Circe.